gdal2mbtiles/gdal.py

```diff
@@ -921,7 +921,7 @@ def get_tempfile(self, **kwargs):
         tempfile.seek(0)
         return tempfile
 
-    def render(self, outputfile, cmd=GDALTRANSLATE, working_memory=512,
+    def render(self, outputfile, cmd=GDALTRANSLATE, working_memory=1024,
```

Comment:
What's the reason for this change in `working_memory`?

Reply:
@mghughes gdal_translate use all the available CPU for rending. so need more memory for processing